src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -5730,6 +5734,9 @@ namespace winrt::TerminalApp::implementation
                 _extensionPalette.ActiveCommandline(L"");
             }
         });
+        _extensionPalette.SetUpProviderInSettingsRequested([&](IInspectable const&, IInspectable const&) {
+            OpenSettingsUI(L"AISettings_Nav");
```

Comment:
A little annoying to hard code this string here - we might want to eventually have an enum for each possible nav page